config/dynamic.c

```diff
@@ -291,6 +292,183 @@ cleanup_dy(struct dynamic_config *dy_conf)
 		rte_free(dy_conf->server_path);
 		dy_conf->server_path = NULL;
 	}
+
+	destroy_mailbox(&dy_conf->mb);
+}
+
+static void
+process_return_message(lua_State *l, struct dynamic_config *dy_conf,
+	int num_succ_sent_inst)
+{
+	int num_gt_messages = 0;
+	size_t reply_len = 0;
+	char reply_msg[MSG_MAX_LEN];
+	int num_cmd_acc = 0;
+
+	/* Wait for all GT instances to synchronize. */
+	while (rte_atomic16_read(&dy_conf->num_returned_instances)
+			< num_succ_sent_inst)
+		rte_pause();
+
+	while (num_cmd_acc < num_succ_sent_inst) {
```

Comment:
We don't need variable `num_cmd_acc`, just use `num_gt_messages` here. It's not just a simplification, there would be a problem if there is a message in the mailbox whose type is not `GT_UPDATE_POLICY_RETURN`.